lib/improver/utilities/cube_metadata.py

```diff
@@ -703,7 +703,7 @@ def extract_diagnostic_name(cube_name):
             The name of the diagnostic underlying this probability
 
     Raises:
-        ValueError: If the input name does not contain 'probability_of_'
+        ValueError: If the input name does not contain 'probability_of'
```

Comment:
This one is an oddity of the sphinx compiler. Basically it cannot cope with trailing slashes, so removing it is the fix I have chosen as I don't think it hides what the error is raised for.